### ISSUE TITLE
Code should be prettified

### ISSUE THREAD

**Issue by jokochems** [pommes-public/pommesdata#46]
There are two useful jupyter notebook extensions (I'm pretty sure there are corresponding jupyterlab extensions as well, but I kind of like the "classical" notebook), namely
* [autopep8](https://pypi.org/project/autopep8/) as well as
* [code_prettify](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/code_prettify/README_code_prettify.html)

which are kind of useful for autoformatting messy code. We should use them to prettify our data preparation script.
Since this will lead to huge and intransparent diffs, it should be included in a dedicated pull request and integrated only after the latest changes (regarding investment modelling plus the necessary fixes for the dispatch data preparation) have been introduced.

**Comment by jokochems** [comment]
@yannickwerner and @hannesfelipe:
* Lately, I've been having troubles with loading jupyter notebook extensions for the particular file `data_preparation.ipynb`.
* It is weird, since the extensions work on any other notebook and also the notebook settings haven't really been changed. Meanwhile, my machine was able to load the extensions again, but this unfortunately was only a one time excemption.
* Of course, this is the major prerequisite to be able to address this issue which would be great to address in the not so far future. 
* It is not that urgent, but could you please have a look, if jupyter notebook extensions, such as the TOC work for you? Thank you!

**Comment by jokochems** [comment]
Closed as this is outdated / a won't fix.
